src/main/java/com/puppycrawl/tools/checkstyle/utils/JavadocUtil.java

```diff
@@ -315,19 +315,10 @@ public static DetailNode getPreviousSibling(DetailNode node) {
      * @throws IllegalArgumentException if an unknown token ID was specified.
      */
     public static String getTokenName(int id) {
```

Comment:
Reused from implementation at https://github.com/checkstyle/checkstyle/pull/10280/commits/4ff4753e7ef94b2a5eb25964ba76315272c50721#r676262572, it makes this a lot simpler.